packages/devextreme/js/__internal/events/core/m_events_engine.ts

```diff
@@ -1,685 +1,695 @@
-import registerEventCallbacks from './event_registrator_callbacks';
-import { extend } from '../../core/utils/extend';
-import { getEventTarget } from '../utils/event_target';
-import domAdapter from '../../core/dom_adapter';
-import { getWindow, hasWindow } from '../../core/utils/window';
+/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable @typescript-eslint/no-dynamic-delete */
```

Comment:
it's better to add ESLint ignore comments to specific parts of the code, rather than applying them to the entire file. I think we can make an exception for the 'max classes per file' comment, which can be used for the whole file